### ISSUE TITLE
Support for Redis HGET, HSET and HDEL

### ISSUE THREAD

**Issue by scozv** [KarelCemus/play-redis#96]
hi @KarelCemus 

Does `play-redis` support `HGET`, `HSET` and `HDEL` right now?

I searched `hget` as keyword in your repo, and I found only `RedisConnectorImpl.scala` contains the `hget` in `hashGet()`.

I tried to inject `RedisConnector`(see below), but in the runtime, I got error:

```
[ExecutionFailedException: Execution of 'HGET mayclass ....' for key 'HGET' failed]
```

My question is does it support `HGET` right now, or I need inject another class?

Or even, I need wait for the implementation, like the implementation of MGET in #81, #82 ?

Thanks

```
import play.api.cache.redis.connector.RedisConnector
import com.google.inject.Inject

class MyClass @Inject() (
  cache: RedisConnector
)(implicit ec: ExecutionContext) {

  def get(key: String): Future[Option[String]] = {
    cache.hashGet("myclass", key)
  }
}
```

**Comment by scozv** [comment]
Sorry, I found my mistake, I forget the `[T]`.

```
def get(key: String): Future[Option[String]] = {
    cache.hashGet[String]("myclass", key)
  }
```

**Comment by KarelCemus** [comment]
Hi, you mention several issues, let me set them straight.

All closed issues and PRs are resolved and published including  #81 and #82, so there is no need to wait for anything but clusters (opened issue).

Do not (ever) inject `RedisConnector`, this is the lowest layer of the `play-redis` architecture. You should be very fine with one of its facade APIs.

A short answer to your question would be "read README", everything is there.

The long answer is that it depends on what API do you use. If `play-redis`'s then yes, it is supported. If Play's then it's not. However, It does not support the `hGet` command directly, it [implements `get` on a `map` instead](https://github.com/KarelCemus/play-redis/blob/master/src/main/scala/play/api/cache/redis/RedisMap.scala#L31). Behavior is the same, but the API looks more consistent and simplified. For more details see [README](https://github.com/KarelCemus/play-redis/blob/master/README.md#how-to-use-this-module) subsection `Example of maps`.

Note: A hash in Redis is basically a map in Scala.

**Comment by scozv** [comment]
> Note: A hash in Redis is basically a map in Scala.

Yes, thanks !

I solved this problem (HGET) with your tips. Now I just inject the `CacheAsyncApi`


```
import play.api.cache.redis.CacheAsyncApi

class MyClass @Inject() (
  cache: CacheAsyncApi
) {
 // your tip works !
}
```
